style.css

```
@@ -0,0 +1,117 @@
+:root {
+    --font-family-heading: "Anton", sans-serif;
+    --font-family-body: "Noto Sans JP", sans-serif;
+    --color-white: #fffefc;
+    --color-black: #020911;
+    --color-accent-red: #f20574;
+    --color-accent-yellow: #f2e52e;
+    --color-background-black: #171525;
+    --color-background-navy: #030427;
+    --color-form-background-yellow: #fffab9;
+    --color-form-placeholder-gray: #ad9783;
+    --color-form-border-red: #df6da2;
+    --color-button-hover-red: #f043a5;
+    --color-course-hover-yellow: #f4e877;
+    --color-hero-overlay: rgba(0, 0, 0, 0.5); /* 背景画像を暗くする */
```

ここ変数定義がfigmaと同じになってないね。右下のcommit suggestionボタン（[参考](https://dev.classmethod.jp/articles/github-multi-line-code-suggestions/)）を押して、commitしてください。

```suggestion
    --color-hero-overlay: rgba(57, 24, 111, 0.4);
```